### PR TITLE
added linked progress plot

### PR DESCRIPTION
**Pull request by williyamshoe**
added linked vertical line that shows up on both progress plots

### CODE REVIEW

File: py/surveyqa/summary.py [hunk 1]
```diff
@@ -160,9 +161,31 @@ def last(arr):
...
+tzone = TimezoneInfo(utc_offset = -7*u.hour)
+t1 = Time(58821, format='mjd', scale='utc')
+t = t1.to_datetime(timezone=tzone)
+line_source = ColumnDataSource(data=dict(x=[t]))
+js = '''
+var geometry = cb_data['geometry'];
+console.log(geometry);
+var data = line_source.data;
+var x = data['x'];
+console.log(x);
```
Comment (sbailey) [inline py/surveyqa/summary.py]: Please remove the leftover debugging `console.log()` calls.

File: py/surveyqa/summary.py [hunk 2]
```diff
@@ -160,9 +161,31 @@ def last(arr):
...
+if (isFinite(geometry.x)) {
+  for (i = 0; i < x.length; i++) {
+    x[i] = geometry.x;
+  }
+  line_source.change.emit();
+}
+'''
```
Comment (sbailey) [inline py/surveyqa/summary.py:185]: Please add a few comment to the above `line_source` and `js` code about what this does and why (target audience: a new collaborator reading this code 2 years from now and wanting to match up the code she is reading to the visual elements on the page).

**Review by sbailey — CHANGES_REQUESTED** [review]
Overall looks good.  Minor requests inline.

**Comment by williyamshoe** [comment]
Alright; I removed the debugging logs and added comments to describe what the js code and the line_source did. I also shortened the js code a bit by removing an unnecessary for loop.

**Review by sbailey — APPROVED** [review]

**Comment by sbailey** [comment]
Looks good; thanks.  Merging now.

**Pull request merged**